Add clarifying documentation for iOS Live Activities `brazeActivityId` (#8961)
braze-inc/braze-docs: _docs/_developer_guide/platforms/swift/live_activities.md

![The BrazeKit framework under Frameworks and Libraries in a sample Xcode project.]({% image_buster /assets/img/swift/live_activities/xcode_frameworks_and_libraries.png %})

#### Step 2.2: Add the BrazeLiveActivityAttributes protocol {#brazeActivityAttributes}

In your `ActivityAttributes` implementation, add conformance to the `BrazeLiveActivityAttributes` protocol, then add the `brazeActivityId` property to your attributes model.

{% alert important %}

> The `brazeActivityId` property will be mapped directly to its corresponding field in the Live Activity push-to-start payload, which is inserted automatically by the Braze sender service. It should not be renamed or assigned to any other value.

Reviewer: ```suggestion
Braze will map the `brazeActivityId` property to the corresponding field in your Live Activity push-to-start payload. It should not be renamed or assigned to any other value.
```

Reviewer: @jerielng , same question here: am i still capturing the essence of your original wording? specifically me removing "inserted automatically by the Braze sender service." if this part is still necessary, do you mind giving a little more conext on what you mean so i can make a cleaner edit? thanks!

Author: It's more that the OS will map the `brazeActivityId` to the field in the payload. What happens here is Braze sends a push notification payload to the device that contains a field called `brazeActivityId`. Integrators must insert this property name into their Swift code and call it the exact same thing, so when the device receives the notification and starts up the live activity, it will see that those two names match and assign the value from the payload into this Swift property.

To verify that your push-to-start notification successfully arrived at the device but was not displayed due to rate limits, you can debug your project using the Console app on your Mac. Attach the recording process for your desired device, then filter the logs by `process:liveactivitiesd` in the search bar.

> #### After receiving a push-to-start notification, why am I not able to receive updates to my Live Activity?

Reviewer: ```suggestion
#### Why isn't the Live Activity updating after receiving a push-to-start notification?
```

Reviewer: @jerielng , made an edit here. just want to verify that i'm still capturing the essence of your original wording? is this still the same question?

Author: Hmm this scenario is more of:
1. I received a push-to-start notification, which successfully starts the live activity on my device -> no issues with push-to-start
2. I attempt to send an update to that live activity
3. That update never shows on my activity.

The scenario is meant to highlight something that only occurs with live activities that were started with a remote notification and not locally from the app, and it's meant to describe only updates failing whereas the start event succeeded.
